Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb

```diff
@@ -0,0 +1,1438 @@
+{
```

Comment:
<p>Could you move</p><blockquote>First, import the <strong>Logistic Regression</strong> module and create a Logistic Regression classifier object using <strong>LogisticRegression()</strong> function. </blockquote><p>to another line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Done